bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v19/BitcoindV19RpcClientTest.scala

```diff
@@ -101,10 +101,11 @@ class BitcoindV19RpcClientTest extends BitcoindRpcTest {
 
     val psbt =
       "cHNidP8BACoCAAAAAAFAQg8AAAAAABepFG6Rty1Vk+fUOR4v9E6R6YXDFkHwhwAAAAAAAA=="
-    val updatedF =
-      clientF.flatMap(client => client.utxoUpdatePsbt(psbt, Seq(descriptor)))
 
-    updatedF.map { result =>
+    for {
+      (client, _) <- clientPairF
```

Comment:
How is this different than the previous version? it seems logically equivalent to me

Reply:
Using `clientPairF` instead of `client` was hoping that might fix.